ANT+_Android_SDK/Sample/ANT+ Plugin Sampler/source/src/main/java/com/dsi/ant/antplus/pluginsampler/pulsezones/Dialog_DataInput.java

```diff
@@ -56,12 +63,18 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         detailsView.findViewById(R.id.radioButton_Zone4).setOnClickListener((View view) -> selectedIdZone = 4);
         detailsView.findViewById(R.id.radioButton_Zone5).setOnClickListener((View view) -> selectedIdZone = 5);
 
-        // Add action buttons
-        builder.setPositiveButton("OK", (DialogInterface dialog, int which) -> {});
 
-                builder.setNegativeButton("Cancel", (dialog, which) -> {
-                    //Let dialog dismiss
-                });
+        //Get shared preferences
+        myPrefs = detailsView.getContext().getSharedPreferences("prefID", Context.MODE_PRIVATE);
```

Comment:
It's good practice to extract as Class constant, strings like this and name it with something self-explanatory such as INTENT_PARAM_APP_PREFERENCES, for instance. The reason ... one single place of this String, cos now you have it on atleast two.